web/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from "react";
+
+import {
+  Button as MuiButton,
+  SxProps,
+  Theme,
+  ButtonProps,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+import { rgbaColors } from "@themes/palette";
+
+interface Props extends ButtonProps {
+  onClick?: () => void;
+  sx?: SxProps;
```

Comment:
Doesn't `ButtonProps` include `sx` already? Maybe pure `ButtonProps` will be enough here?